examples/chip-tool/templates/commands.zapt

```diff
@@ -145,11 +138,24 @@ static void On{{asCamelCased parent.name false}}{{asCamelCased name false}}ListA
         {{#if isStruct}}
         ChipLogProgress(chipTool, "{{type}}[%lu]:", i);
         {{#chip_attribute_list_entryTypes}}
+        {{#if (isOctetString type)}}
+        ChipLogProgress(Zcl, "  {{asSymbol label}}: %zu", entries[i].{{name}}.size());
+        {{else if (isCharString type)}}
+        // Currently the generated code emits `uint8_t *` for CHAR_STRING, it needs to emits chip::ByteSpan
+        // ChipLogProgress(Zcl, "  {{asSymbol label}}: %.*s", entries[i].{{name}}.size(), entries[i].{{name}}.data());
+        {{else}}
         ChipLogProgress(chipTool, "  {{name}}: {{asPrintFormat type}}", entries[i].{{name}});
+        {{/if}}
         {{/chip_attribute_list_entryTypes}}
         {{else}}
+        {{#if (isOctetString type)}}
+        ChipLogProgress(Zcl, "  {{asSymbol label}}: %zu", entries[i].size());
```

Comment:
`%zu` is not available on all embedded platforms. 

*In a future PR, suggest:*

Suggest having the code eventually static cast size() to `uint32_t` in the log call and use `PRIu32` for the specifier.